src/sqs/index.ts

```diff
@@ -351,7 +358,7 @@ export class SqsQueue<MessagePayload> {
 					(errorHandlingResult.skipDlq && context.receiveCount >= this.maxAttempts)) {
 					log.info("Deleting the message hence it reached the maximum amount of retries")
 					await this.deleteMessage(message, log)
-				} else if (errorHandlingResult.retryDelaySec) {
+				} else if (errorHandlingResult.retryDelaySec !== undefined /*zero delay supported*/) {
```

Comment:
nit: it took me 15-20 seconds to digest the comment. Might something like `"zero seconds delay is also supported" ` be more clear?